### DISCUSSION TITLE
scedule on public page

### DISCUSSION THREAD

**Discussion by burnin-hell** [AzuraCast/AzuraCast#6196] [Q&A]
is it posible to show the scedule in al list view ( just like on the adminpage) on a public page ? now it is shown as a calender view what is not nice on a website or is there a api call to do this ?

**Comment by Mozitux** [comment]
I'm not sure about what you're wondering, but you can display the scheduled page as a widget using iframe.

**Reply by burnin-hell** [reply]
I know but thats A calender view I want A list just like on the admin page as widget

**Comment by Moonbase59** [comment]
@burnin-hell: I think you mean a display like the "Scheduled" view on the station’s "Profile" page? You can easily achieve that using the API (plus a little programming to make it look nice). Read the documentation at https://your.station.com/static/api/index.html#/Stations%3A%20Schedules/getSchedule then try something like the following (assuming station \# 1):

```
https://your.station.com/api/station/1/schedule
```

You can even give it a starting time (2023-04-23T00:00:00) and \# rows to return (10):

```
https://your.station.com/api/station/1/schedule?now=2023-04-23T00%3A00%3A00&rows=10
```

You’ll get a nice clean JSON response that you can use to display your requested schedule in whichever way you want:

![Auswahl_205](https://user-images.githubusercontent.com/3706922/233834876-e38d07cd-41c9-4de1-a10f-bedf7baa1bc9.png)

HTH. Good luck!

N.B.: Note that in the current rolling release (Rolling Release #[c0e9752](https://github.com/AzuraCast/AzuraCast/commit/c0e975294ef2b80f83cea4c107c4d296b516a081) (2023-04-22 9:11)) `is_now` isn’t correctly set to `true` when using the "start time" method. I filed an issue at https://github.com/AzuraCast/AzuraCast/issues/6231.
